lib/parallel.php

```diff
@@ -12,14 +12,20 @@
 $procs = array();
 
 foreach ($CONFIG['environments'] as $key => $value) {
-    $cmd = "TASK_ID=$key ./bin/behat --config=". getenv("CONFIG_FILE")." 2>&1\n";
+    if(stripos(PHP_OS, 'WIN') === 0) {
```

Comment:
will PHP_OS return win32 / win64 if yes `preg_match('/win/i', PHP_OS)`?

Reply:
possible values for PHP_OS on windows systems include 
- WIN32
- WINNT
- Windows
ref: https://github.com/php/php-src/blob/0d51ebd1a54af59d915c551a240b56bb3f0e26a6/main/main.c#L2000